src/Codeception/Module/WebDriver.php

```diff
@@ -3157,4 +3158,16 @@ protected function setBaseElement($element = null)
         }
         $this->baseElement = $this->matchFirstOrFail($this->webDriver, $element);
     }
+
+    protected function enableImplicitWait($enable)
```

Comment:
Using Boolean arguments is a bad practice, please split this method to `enableImplicitWait` and `disableImplicitWait`.